source/linux/Foundational_Components/Hypervisor/Jailhouse.rst

```diff
@@ -399,7 +393,7 @@ the demo are:
 
     .. code-block:: console
 
-        root@am62xx-evm: jailhouse enable /usr/share/jailhouse/cells/k3-am625-sk.cell
+        root@<machine>: jailhouse enable /usr/share/jailhouse/cells/k3-am625-sk.cell
```

Comment:
A handful of these blocks are inherently machine specific. I do not think that context should be removed.

Reply:
@StaticRocket Well that was intentional. Apparently AM62X includes am62xxsip-evm, am62xx-evm and am62xx-lp-evm. That's the reason @paresh-bhagat12 generalized it as machine as jailhouse is currently supported on both am62xx-evm and am62xx-lp-evm

Reply:
Those devices don't have a different prompt though, do they?

Reply:
@StaticRocket it's they have different prompt :wink: - https://git.yoctoproject.org/meta-ti/tree/meta-ti-bsp/conf/machine?h=scarthgap